laser_encoders/language_list.py

```diff
@@ -13,6 +13,34 @@
 # -------------------------------------------------------
 # Language mapping to handle different language codes and names
 
+
+def build_language_names_dict(language_list, language_names):
+    result_dict = {}
+
+    for lang_code in language_list:
+        if lang_code in language_names:
```

Comment:
If `lang_code` is not in `language_list`, let's raise an error, because either the code is misspelled or the list is incomplete; in either case, we would like to find and fix the problem